experiments/integrated/fluxnet/fluxnet_simulation.jl

```diff
@@ -1,7 +1,7 @@
 """This file contains the site-generic time variables for running ClimaLand on
 fluxtower sites. These work in tandem with the site-specific timing parameters
 found in the {site-ID}_simulation.jl files in each site directory."""
-
+data_dt = Float64(FluxnetSimulationsExt.get_data_dt(site_ID))
```

Comment:
can you define this in `run_fluxnet.jl` instead? Thanhthanh's PR is removing this file

Reply:
I will call it from run_fluxnet instead.